lib/ansible/cli/scripts/ansible_connection_cli_stub.py

```diff
@@ -99,7 +99,7 @@ def start(self, variables):
                 self.play_context.private_key_file = os.path.join(self.original_path, self.play_context.private_key_file)
             self.connection = connection_loader.get(self.play_context.connection, self.play_context, '/dev/null',
                                                     task_uuid=self._task_uuid, ansible_playbook_pid=self._ansible_playbook_pid)
-            self.connection.set_options(var_options=variables)
+            self.connection.set_options(direct=variables)
```

Comment:
this change is not needed, but also not a problem since there would only be a precedence issue if you had both